brands/shop/electronics.json

```diff
@@ -146,6 +146,15 @@
       "shop": "electronics"
     }
   },
+  "shop/electronics|Gigantti": {
```

Comment:
should there be a country code here?

Reply:
My script will autoadd country codes, manually specifying country codes. It will work well in cases where OSM data is describing real world well (each country where this brand is mapped has some elements mapped). Assuming that this brand actually appears only in Finland, then it will work here - http://overpass-turbo.eu/s/Jw9

Reply:
@crimsondusk It is Finland only brand, right? (if not - adding country code is certainly useful, for Finland only you can either add country code or wait for my script to do it).

Reply:
Looks like I forgot to add the country code here.